web/src/.eslintrc.js

```diff
@@ -34,7 +34,6 @@ module.exports = {
     'react/jsx-indent': 'off',
 
     // typescript-eslint rules
-    // TODO: add options { allowExpressions: true, allowTypedFunctionExpressions: false }
     '@typescript-eslint/explicit-function-return-type': 'off',
```

Comment:
Can we invert this? Set it to what we want, but then override `.js` files to turn it off instead?

Once we're 100% typescript we can remove the override too that way.